### ISSUE TITLE
Fix Linux Console Window buffer size

### ISSUE THREAD

**Issue by pps83** [Microsoft/VSLinux#162]
Please, don't set retarded 10K limits on output buffer that shows in "Linux Console Window".
I just run a test for 45 minutes that logs a line each second and sure enough I wasted time waiting for the test to end, because I was naive thinking that in 2017 it should be technically possible to store a couple of megs of output

**Comment by stanthomas** [comment]
Err, why not redirect output to a file? You surely can't read that much information as is shoots by in a console window.

**Comment by pps83** [comment]
VS debug output has not limits (at least Ive never seen it). Same for the console output from remote process. There is no need to limit it, or at least keep full output log in Debug IntDir just like regular VS does.

**Comment by itodirel** [comment]
Thanks for the suggestion, easy to fix, we'll change the default to something more sensible and provide an override for users. The current buffer is larger than 10k.

**Comment by pps83** [comment]
In my case it was exactly 10K. Regular dev console in VS has no limits IMO, or at least I've never hit

**Comment by stanthomas** [comment]
@lukka
I do not agree with your perfunctory closing of this issue. Regardless of the merit of the issue raised, there is an outstanding action against the development team (@itodirel) and you should report on that first. Either 'adressed in release xx.xx by *change detail*' **or** 'will not be actioned because *reason*'. The passage of time is irrelevant; it is quite possible that others with a similar issue have found this topic and concluded that a resolution is on its way.

**Comment by itodirel** [comment]
@lukka can we reproduce this, is it a bug?
